jenkins/opensearch/publish-min-snapshots.jenkinsfile

```diff
@@ -52,72 +52,72 @@ pipeline {
         }
         stage('Build OpenSearch-min Snapshots') {
             parallel {
-                stage('linux-x64-tar') {
-                    agent {
-                        docker {
-                            label AGENT_LINUX_X64
-                            image dockerAgent.image
-                            args dockerAgent.args
-                            registryUrl 'https://public.ecr.aws/'
-                            alwaysPull true
-                        }
-                    }
-                    steps {
-                        script {
-                            buildManifest(
-                                componentName: "OpenSearch",
-                                inputManifest: "manifests/${INPUT_MANIFEST}",
-                                platform: 'linux',
-                                architecture: 'x64',
-                                distribution: 'tar',
-                                snapshot: true
-                            )
-                            echo("Uploading linux x64 tar min snapshots to S3")
-                            uploadMinSnapshotsToS3(
-                                fileActions: [createSha512Checksums()],
-                                distribution: 'tar'
-                            )
-                        }
-                    }
-                    post {
-                        always {
-                            postCleanup()
-                        }
-                    }
-                }
-                stage('linux-arm64-tar') {
-                    agent {
-                        docker {
-                            label AGENT_LINUX_ARM64
-                            image dockerAgent.image
-                            args dockerAgent.args
-                            registryUrl 'https://public.ecr.aws/'
-                            alwaysPull true
-                        }
-                    }
-                    steps {
-                        script {
-                            buildManifest(
-                                componentName: "OpenSearch",
-                                inputManifest: "manifests/${INPUT_MANIFEST}",
-                                platform: 'linux',
-                                architecture: 'arm64',
-                                distribution: 'tar',
-                                snapshot: true
-                            )
-                            echo("Uploading linux arm64 tar min snapshots to S3")
-                            uploadMinSnapshotsToS3(
-                                fileActions: [createSha512Checksums()],
-                                distribution: 'tar'
-                            )
-                        }
-                    }
-                    post {
-                        always {
-                            postCleanup()
-                        }
-                    }
-                }
+                //stage('linux-x64-tar') {
```

Comment:
Why do we want to comment out all of these stages?

Reply:
Test only, removed.